invoices/models.py

```diff
@@ -10,7 +10,7 @@
 
 
 def is_phase_billable(phase_name, project):
-    if project in ("[Leave Type]", "Solinor"):
+    if project in ("[Leave Type]", "LeaveType", "Solinor"):
```

Comment:
Wouldn't it make more sense to have this as a constant, or as a separate field, now that the list is growing again? Or at least normalizing `[Leave Type]` and `LeaveType`?

Reply:
Because that method seems to be used only from importing which always get data from same API then data is always in same format, so `[Leave Type]` could be removed.